tests/Example/UploadHandler.php

```
@@ -0,0 +1,77 @@
+use function basename;
+use function is_dir;
+use function mkdir;
+
+/**
+ * Example upload handler implementation
+ *
+ * This is a reference implementation showing how to use FileUpload in your application.
+ */
+final class UploadHandler
+{
+    public function __construct(
```

PHPStan (2.1.31)

Method Koriym\FileUpload\Example\UploadHandler::__construct() has parameter $allowedTypes with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)